### DISCUSSION TITLE
Welcome to mobile Discussions!

### DISCUSSION THREAD

**Discussion by veloce** [lichess-org/mobile#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jonbgamble** [comment]
Let us write bad dart and misuse the flutter framework for the glory of lichess.

**Comment by jonbgamble** [comment]
Many are itching to get started.  This is a good place to discuss app development prior to putting everyone's pet feature requests in GitHub issues.

Veloce needs to get some basic structure and functionality in place - noone wants the app to grow like a cancer.  This means it could be awhile before we see a first commit to the official repo.  Be patient, star the repo, and hone your skills because Lichess mobile v2 is coming and we're all gonna help make it!

**Reply by luisgurmendez** [reply]
@ornicar first commit msg should have been `e4`, is there a chance we can create a new repo? /s

**Comment by michalsrutek** [comment]
@veloce Do you have a specific state management (framework) in mind yet?

**Reply by veloce** [reply]
Yes I'm gonna try Riverpod.

**Reply by malaschitz** [reply]
> Yes I'm gonna try Riverpod.

Great ! I am working on my chess app with Riverpod too...

**Comment by malaschitz** [comment]
I found another small error in the https://pub.dev/packages/bishop 

Will a custom library for chess logic and display be created for lichess, or will something existing be used (like the bishop one) ?

**Comment by egecavusoglu** [comment]
Hey folks, appreciate the work you do! What a nice initiative to develop the lichess mobile app open sourced. As a React Native fan, was wondering if you have considered it and what was the decision behind not using it. Again, not criticizing just curious about the decision process. 🙂

I believe a lot of the previous JS/TS logic could have been reused, as well as chess libs/tooling built in the JS ecosystem.

Cheers.

**Comment by HappyTreesDev** [comment]
I will say as a person who has written apps for production in both react native and flutter, they both get the job done, but I have found flutter much more enjoyable in general to work in.

But in reality, these things often come down to using the tool you know really.
